CHANGES.rst

```diff
@@ -2,8 +2,10 @@
 Changelog
 =========
 
-Version 3.2.1 (2019-XX-XX)
-==========================
+Next version (Unreleased)
+=========================
+
+- Fix an issue where a segmentation fault may be raised when comparing MetaPartition instances
```

Comment:
FYI: the issue is fixed in pyarrow 0.14.1 which is out now. I would blacklist 0.14.0 and only allow 0.14.1, so the description needs to be adjusted here.

Reply:
I don't why we would need to adjust this. There is no arrow 0.14.1 compatible Kartothek version

Reply:
So, I'm just wondering what the changelog should read :) I don't have a strong opinion here

Reply:
If we don't support `0.14.0` and will just finish #108 to support `0.14.1` (which still blacklists `0.14.0`), I think from a user perspective "Add support for pyarrow 0.14.0" would be a valid changelog entry.

Reply:
This isn't about arrow versions. The issue is fixed in 0.14.1. but still exists in all other versions. The reason why we don't want to support 0.14.0 is a completely different topic and has nothing to do with this segfault

Reply:
Ah I remember, this is actually also buggy in older versions, right? Then replace `segfault` with `potential segfault` and then this should be OK.